doc/vpi/examples/quickstart.rst

```diff
@@ -0,0 +1,17 @@
+.. program:: ghdl
+.. _COSIM:VPI:Examples:quickstart:
+
+Quick Start
+###########
+
+.. _COSIM:VPI:Examples:quickstart:hello:
+
+:cosimtree:`hello <vpi/quickstart/vpi_hello.c>`
+***********************************************
+
+This is the most minimal example, where a single callback is registered at the beginning of the simulation. The callback just
+prints ``Hello!``. Then, the simulation is executed as usual.
+
+VPI allows to register callbacks at multiple events and to optionally delay their execution after the event is triggered.
```

Comment:
allows registering/allows one to register

Reply:
Should it be "and optionally delaying" too?